public/app/plugins/panel/barchart/utils.ts

```diff
@@ -80,7 +80,7 @@ export const preparePlotConfigBuilder: UPlotConfigPrepFn<BarChartOptions> = ({
 
   builder.addHook('init', config.init);
   builder.addHook('drawClear', config.drawClear);
-  builder.addHook('draw', config.draw);
+  builder.addHook('drawPoints', config.drawPoints);
```

Comment:
@leeoniya TS is complaining about drawPoints missing from hook defs - has it been removd or we ts-ignored when we used it some time ago?

Reply:
there's no `drawPoints` hook. drawPoints is just a function to use for `series.points.show`